providers/implementations/digests/sha3_prov.c

```diff
@@ -177,11 +177,18 @@ static int s390x_shake_final(unsigned char *md, void *vctx)
     return 1;
 }
 
+size_t SHA3_absorb(uint64_t A[5][5], const unsigned char *inp, size_t len,
+                   size_t r);
+
+void SHA3_squeeze(uint64_t A[5][5], unsigned char *out, size_t len, size_t r);
+
```

Comment:
Why are these function prototype declarations needed?

Reply:
Sorry, they are left-overs from a previous fix that did not use CPACF in this case.  They are not needed.  I removed them.